aws/table_aws_drs_source_server.go

```diff
@@ -0,0 +1,199 @@
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "data_replication_info",
+				Description: "The Data Replication Info of the Source Server.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "last_launch_result",
+				Description: "The status of the last recovery launch of this Source Server.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "life_cycle",
+				Description: "The lifecycle information of this Source Server.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "replication_direction",
+				Description: "Replication direction of the Source Server.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "reversed_direction_source_server_arn",
+				Description: "For EC2-originated Source Servers which have been failed over and then failed back, this value will mean the ARN of the Source Server on the opposite replication direction.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "source_cloud_properties",
+				Description: "Source cloud properties of the Source Server.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "staging_area",
+				Description: "The staging area of the source server.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "staging_account_id",
+				Description: "The staging account ID that extended source servers belong to.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("StagingArea.StagingAccountID"),
+			},
+			{
+				Name:        "hardware_id",
+				Description: "An ID that describes the hardware of the Source Server. This is either an EC2 instance id, a VMware uuid or a mac address.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromQual("hardware_id"),
+			},
+
+			// Steampipe standard columns
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("SourceServerID"),
+			},
+			{
+				Name:        "tags",
+				Description: resourceInterfaceDescription("tags"),
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Tags"),
+			},
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Arn").Transform(arnToAkas),
+			},
+		}),
+	}
+}
+
+//// LIST FUNCTION
+
+func listAwsDRSSourceServers(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+
+	// Create service
+	svc, err := DRSClient(ctx, d)
+	if err != nil {
+		plugin.Logger(ctx).Error("aws_drs_source_server.listAwsDRSSourceServers", "connection_error", err)
+		return nil, err
+	}
+
+	if svc == nil {
+		// Unsupported region, return no data
+		return nil, nil
+	}
+
+	maxItems := int32(10000)
```

Comment:
Is 10,000 the max limit for this? The API doc doesn't seem to mention a limit, so can it go beyond 10,000?